signin.js

```diff
@@ -0,0 +1,79 @@
+if (localStorage.getItem("accessToken")) {
+    location.replace("/folder");
+}
+
+const eInput = document.getElementById('email');
+const pInput = document.getElementById('password');
+const emailErrMsg = document.getElementById('email-err-msg');
+const passwordErrMsg = document.getElementById('password-err-msg');
+const signinbtn = document.querySelector('#signin-btn');
+const eChecker = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-za-z0-9\-]+/; 
+
+
+function emailCheck(email) {
+    if(eChecker.test(email) === false) {
+        return false; 
+    } else { 
+        return true; 
+    }
+}
+
+function printEmailMsg(e){
+    emailErrMsg.textContent = '';
+    if(eInput.value == ''){
+        emailErrMsg.textContent = '이메일을 입력해주세요.';
+        return false;
+    } else {
+        if(emailCheck(eInput.value) == false) {
+            emailErrMsg.textContent = '올바른 이메일 주소가 아닙니다.';
+            return false;
+        }
+    }
+    return true;
```

Comment:
[중요도: 높음] 로직 자체는 문제가 없어 보이나 가독성에 조금 문제가 있어보입니다.
```suggestion
    if(eInput.value == ''){
        emailErrMsg.textContent = '이메일을 입력해주세요.';
        return false;
    }
    if(emailCheck(eInput.value) == false) {
            emailErrMsg.textContent = '올바른 이메일 주소가 아닙니다.';
            return false;
    }
    return true;
```

이게 더 좋아보이네요.

[중요도: 중간] 함수명이 조금 이상합니다. printEmailMsg보다는 checkValidEmail 같은 형태가 나아보입니다.
해당 함수의 주 목표는 이메일 메세지 표현보다는 이메일 사용이 가능한지를 체크하는게 더 주요한 것 같네요.